docs/templates/override/styles/override.css

```diff
@@ -8,6 +8,12 @@ body {
     font-size: 16px;
 }
 
+.affix ul ul > li > a:before {
+    font-size: 16px;
+    top: 0px;
+    left: 3px;
+}
+
```

Comment:
This was a slightly unrelated change, but it turned out the affix-TOC was rendered with a `>` character which was rendered with the wrong font size. The position was also slightly tweaked to make it look better.